@navikt/core/react/src/util/hooks/descendants/useDescendant.tsx

```diff
@@ -74,7 +74,7 @@ export function createDescendantContext<
    * @internal
    * Initializing DescendantsManager
    */
-  function _useDescendants() {
+  function useDescendants() {
```

Comment:
Btw, why are they marked as internal?

Reply:
Good question 🤔 Since they are nit exposed in util-dir, they can only be used by us anyways..